src/main/java/br/com/caelum/viagens/administrativo/controller/dto/input/NewAeroportoInputDto.java

```diff
@@ -0,0 +1,42 @@
+package br.com.caelum.viagens.administrativo.controller.dto.input;
+
+import java.util.Optional;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import br.com.caelum.viagens.administrativo.controller.dto.PossuiPaisDto;
+import br.com.caelum.viagens.administrativo.model.Aeroporto;
+import br.com.caelum.viagens.administrativo.model.Pais;
+import br.com.caelum.viagens.administrativo.repository.PaisRepository;
+
+public class NewAeroportoInputDto implements PossuiPaisDto {
+
+	@NotBlank
+	private String nome;
+
+	@NotNull
+	private Long paisId;
```

Comment:
Deixa eu perguntar uma coisa... Já que você usa o lance do Converter para objetos de dominio lá no controller, pq não usa aqui também? Ia poupar receber um Repository como argumento e dar aquele get.

Reply:
Segundo a documentação do Spring Data, "DomainClassConverter to let Spring MVC resolve instances of repository-managed domain classes from request parameters or path variables." - ou seja, pelo que eu entendi, eu não consigo usar com @ResponseBody. Sua sugestão seria fazer esperando que o cliente mandaria o "paisId" de outra  maneira e usar  @PathVariable ou @RequestParam?